Book/php7/extensions_design/extension_skeleton.rst

```diff
@@ -115,8 +115,7 @@ The ``ini_entry`` vector is actually unused. You :doc:`register INI entries <ini
 
 Then you may declare dependencies, that means that your extension could need another extension to be loaded before it,
 or could declare a conflict with another extensions. This is done using the ``deps`` field. In reality, this is very
-ucommonly used, and more generally it is a bad practice to create dependencies across PHP extensions. Also, note that
-dependencies are
+ucommonly used, and more generally it is a bad practice to create dependencies across PHP extensions.
```

Comment:
I was about to submit the same fix. Also note that `ucommonly` should be `uncommonly`

Reply:
@TysonAndre thank you for pointing it out. I have added another commit in this PR to fix it.